ert3/console/_console.py

```diff
@@ -122,6 +122,10 @@ def _build_status_argparser(subparsers: Any) -> None:
     subparsers.add_parser("status", help="Report the status of all experiments")
 
 
+def _build_visualise_argparser(subparsers: Any) -> None:
```

Comment:
Why not `visualize` ? :thinking:

Reply:
I changed the command to `ert3 vis` for it to be consistent with `ert vis`